include/invert_quda.h

```diff
@@ -673,6 +733,70 @@ namespace quda {
     virtual bool hermitian() { return true; } /** CG is only for Hermitian systems */
   };
 
+  /**
```

Comment:
I think this should be pulled out in its own header

Reply:
https://github.com/lattice/quda/pull/1061/commits/96926239a741adb4fa5c8bab17dc6714e094af3b